cmd/nerdctl/login.go

```diff
@@ -253,6 +253,12 @@ func tryLoginWithRegHost(ctx context.Context, rh docker.RegistryHost) error {
 		Host:   rh.Host,
 		Path:   rh.Path,
 	}
+	if rh.Path == "/v2" {
+		// If the path is using /v2 endpoint but lacks trailing slash add it
+		// https://docs.docker.com/registry/spec/api/#detail. Acts as a workaround
+		// for containerd issue https://github.com/containerd/containerd/blob/main/remotes/docker/config/hosts.go#L110
```

Comment:
> https://github.com/containerd/containerd/blob/main/remotes/docker/config/hosts.go#L110

Please use a permanent link with the commit hash (or just the tag), e.g., 
https://github.com/containerd/containerd/blob/2986d5b077feb8252d5d2060277a9c98ff8e009b/remotes/docker/config/hosts.go#L110

Reply:
Fixed